examples/user.py

```diff
@@ -34,36 +34,40 @@
     'mobilePhone': '555-2121',
     'officePhone': '555-1212',
     'password': 'myPass1234',
-    'roles': ['Read only'],
-    'type': 'UserAndRoles',
+    'permissions': [
+        {
+            'roleName': 'Read only',
+            'scopeUri': '/rest/scopes/b7259c1c-c52a-4be3-82de-d50e1730695e',
+        }
+    ],
+    'type': 'UserAndPermissions',
     'userName': 'testUser'
 }
 
 # Try load config from a file (if there is a config file)
 config = try_load_from_file(config)
 
 oneview_client = OneViewClient(config)
+users = oneview_client.users
 
 # Create a User
-user = oneview_client.users.create(options)
-print("Created user '%s' successfully.\n  uri = '%s'\n" % (user['userName'], user['uri']))
+user = users.create(options)
+print("Created user '%s' successfully.\n  uri = '%s'\n" % (user.data['userName'], user.data['uri']))
 
-# Adds different roles to the recently created user
-user = oneview_client.users.get_by('userName', 'testUser')
-print(user)
-print("\n user: %s, roles before update: %s" % (user['userName'], user['roles']))
-user['replaceRoles'] = True
-user['roles'] = ['Infrastructure administrator']
-user = oneview_client.users.update(user)
-print("\n user: %s, roles after update: %s" % (user['userName'], user['roles']))
-
-# Get user by role
-user = oneview_client.users.get_by('role', 'Read only')
-print("Found users by role: '%s'.\n '\n" % (user))
+# Change Password
+change_password_request = {
+    "currentPassword": "admin12345",
+    "enabled": "true",
+    "password": "admin1234",
+    "userName": "admin"
+}
+changePasswordResponse = users.change_password(change_password_request)
+print("Changed Password successfully")
+print(changePasswordResponse)
 
 # Get user by name
-user = oneview_client.users.get_by('userName', 'testUser')
-print("Found user by name: '%s'. uri = '%s'\n" % (user['userName'], user['uri']))
+user = oneview_client.users.get_by_userName(options['userName'])
```

Comment:
Since you already created 'users' object, you can call all methods using that object.
```
users_by_name = users.get_by_userName(options['userName'])
```

Reply:
done